config/motoros2_config.yaml

```diff
@@ -70,6 +70,33 @@
 # DEFAULT: true
 sync_timeclock_with_agent: true
 
+#-----------------------------------------------------------------------------
+# Should MotoROS2 monitor the link state of the ethernet port used to
+# communicate with the Agent?
+#
+# If enabled, this will cause immediate notification of loss-of-connection
+# when the link goes down, causing MotoROS2 to activate it's shutdown
+# behaviour (ie: stop all motion (if enabled), release resources, wait for
+# link up and attempt reconnection with the Agent).
+#
+# If this is disabled, Agent disconnection detection uses an application
+# layer based timeout mechanism which takes longer to detecet disconnects (but
+# might be more robust against intermittent link-layer disconnects which do not
+# cause network disconnections).
+#
+# DEFAULT: true
+#userlan_monitor_enabled: true
+
+#-----------------------------------------------------------------------------
+# Which port should MotoROS2 monitor, if 'userlan_monitor_enabled' is 'true'?
```

Comment:
I really believe we should automatically detect this. It's nicer for the user.

Plus, a future robot controller could _potentially_ have additional ports which would confuse the issue more.

Reply:
Bah. Auto-detecting means more logic to add :(

But ok, I'll see if I have time to look into it.

Reply:
I believe I've implemented something in this direction.

It's pretty naive though: it will just check whether the Agent IP is reachable on a particular interface and assume that's the one it should monitor.